content/src/content/jcr_root/apps/core/wcm/components/search/v1/search/clientlibs/site/js/search.js

```diff
@@ -207,11 +207,6 @@
         var self = this;
 
         switch (event.keyCode) {
-            case keyCodes.TAB:
```

Comment:
This seems to have been intentionally removed before. We should make sure it does not interfere with results navigation (up-down arrows) and tab selection and up-down results selection look and behave consistently.